core/manifest/ClassLoader.php

```diff
@@ -101,6 +101,7 @@ public function getItemPath($class) {
 	 * @return bool
 	 */
 	public function classExists($class) {
+		Deprecation::notice('3.3', 'Not necessary anymore as this function is inlined in \'ClassInfo::exists\' for speed and that\'s the only place where it is used in core.');
```

Comment:
I don't really see the appeal of PSR-4 so I don't completely understand it's purpose. How do my changes not work well with PSR-4?